src/ticketing/types.ts

```diff
@@ -126,33 +128,32 @@ export type ReserveOfferRequest = {
    * Recurring payment id should be provided if using a previously stored
    * payment card
    */
-  recurring_payment_id?: number;
+  recurringPaymentId?: number;
   /**
    * Payment type and the store payment flag should be provided if not using a
    * previously stored payment card
    */
-  payment_type?: PaymentType;
-  store_payment?: boolean;
+  paymentType?: PaymentType;
+  storePayment?: boolean;
   /**
    * Paying customer's phone number, with country prefix. The phone number and
    * redirect URL is only required for mobile payments types, e.g. Vipps.
    */
-  phone_number?: string;
-  payment_redirect_url?: string;
-  sca_exemption?: boolean;
+  phoneNumber?: string;
+  paymentRedirectUrl?: string;
+  scaExemption?: boolean;
   /** Only needed if fare contract should be created on a different account */
-  customer_account_id?: string;
+  customerAccountId?: string;
   /**
    * Only needed if fare contract should be created on a different account
    * which should be saved.
    */
-  store_alias?: {
+  storeAlias?: {
     alias: string;
     /** With country prefix */
-    phone_number: string;
+    phoneNumber: string;
   };
-  /** Experimental */
-  auto_sale?: boolean;
+  autoSale?: boolean;
```

Comment:
Have tried to mirror backend comments here. Also a URL to Nets docs could be useful ✨ 

```suggestion
  /**
   * Uses auto sale for Nets. Defaults to false.
   * https://developer.nexigroup.com/netaxept/en-EU/api/rest-v1/#netaxept-registeraspx-get-parameters-autosale
   */
  autoSale?: boolean;
```